apps/class-solid/src/components/plots/LinePlot.tsx

```diff
@@ -32,28 +30,3 @@ function Line(d: ChartData<Point>) {
     </path>
   );
 }
-
```

Comment:
You spelling out the line plots in Analysis.tsx instead of reusing a component. 

Why? Where there too many properties? It it more readable?

Reply:
I had reduced the lineplot to its bare basics, and then I felt I didn't really need the separate component anymore.

Reply:
ok